cmd/lachesis/commands/run.go

```diff
@@ -121,6 +130,9 @@ func AddRunFlags(cmd *cobra.Command) {
 	cmd.Flags().String("datadir", config.Lachesis.DataDir, "Top-level directory for configuration and data")
 	cmd.Flags().String("log", config.Lachesis.LogLevel, "debug, info, warn, error, fatal, panic")
 	cmd.Flags().Bool("log2file", config.Log2file, "duplicate log output into file lachesis_<BindAddr>.log")
+	if runtime.GOOS != "windows" {
```

Comment:
It is not enough, "github.com\sirupsen\logrus\hooks\syslog" has "// +build !windows,!nacl,!plan9" constraint.

Reply:
Corrected. Thanks.

Reply:
Corrected incorrect :(
Don't ignore requested changes above (by me). They are necessary, else we'll get compilation error on Windows.

Reply:
https://github.com/Fantom-foundation/go-lachesis/pull/123/files/624d752aa44a25803f3f05e85a28d3ffe24a7ff4#r244718589

@devintegral I agree that `// +build !windows,!nacl,!plan9` is the build constraint introduced by `github.com/sirupsen/logrus/hooks/syslog`. However, if the `github.com/sirupsen/logrus/hooks/syslog` build constraint changes, then we need to update our build constraint. This creates a dependency between explicitly specified build constraints, which are unnecessary, as these are already implicitly enforced by Go through its module system.

If you want to introduce all build constraints set by other packages explicitly, then one will need to create a separate issue requiring investigation of the build constraints of all dependent packages into the first line of each file that depends on them.

Reply:
@Maxime2, great idea! Stub syslog hook would be the best solution. I've contributed it: github.com/sirupsen/logrus/pull/883 . But we shoud use build constraints here until merged.

Reply:
@devintegral github.com/sirupsen/logrus/pull/883 is very raw yet, in particular, copying Priority constants from sys/syslog package is a bad idea.
I have implemented our custom logrus hook which uses either syslog, either event log when appropriate or uses stub for nacl and plan9.

Reply:
@Maxime2, you suggest hardcoded syslog.Priority instead of copying constants? Hm... maybe

Reply:
Nope, I mean priority argument for NewSyslogHook() is not needed at all and thus it is not needed  to bring those constants out of build tag in sys/syslog package. See how it's done in our custom hook.